modules/fx/arrow-fx-rx2/src/test/kotlin/arrow/fx/ObservableKTests.kt

```diff
@@ -61,7 +63,8 @@ class ObservableKTests : RxJavaSpec() {
       TraverseLaws.laws(ObservableK.traverse(), ObservableK.functor(), { ObservableK.just(it) }, EQ()),
       ConcurrentLaws.laws(CO, EQ(), EQ(), EQ(), testStackSafety = false),
       TimerLaws.laws(ObservableK.async(), ObservableK.timer(), EQ()),
-      FunctorFilterLaws.laws(ObservableK.functorFilter(), { Observable.just(it).k() }, EQ())
+      FunctorFilterLaws.laws(ObservableK.functorFilter(), { Observable.just(it).k() }, EQ()),
+      MonadFilterLaws.laws(ObservableK.monadFilter(), { Observable.just(it).k() }, EQ())
```

Comment:
MonadFilterLaws should include FunctorFilterLaws, so there's no need to have both here.